bin/faucet/Cargo.toml

```diff
@@ -32,7 +32,11 @@ miden-tx = { workspace = true,  features = ["concurrent"] }
 rand = { version = "0.8" }
 rand_chacha = "0.3"
 serde = { version = "1.0", features = ["derive"] }
+static-files = "0.2.1"
```

Comment:
nit: we usually don't specify patch versions for dependencies (i.e., it would be `0.2` instead of `0.2.1`).